src/Compat/CompatForm.php

```diff
@@ -22,4 +48,24 @@ public function hasDefaultElementDecorator()
 
         return true;
     }
+
+    /**
+     * Returns a duplicate submit element with an added `class` attribute
```

Comment:
This sentence needs to be corrected several times:

1. `Return` vs `Returns`.
2. It doesn't  add a `class` attributes, it overrides it with a specific value.
3. We're talking about `submit button` not `submit element` throughout this PR.

Consider rephrasing to:

> Return a duplicate of the given submit button with the `class` attribute fixed to `primary-submit-btn-duplicate`